metaflow/util.py

```diff
@@ -339,3 +340,22 @@ def _access_check(fn, mode):
                         return name
 
         return None
+
+
+def to_pascalcase(obj):
+    """
+    Convert all keys of a json to pascal case.
+    """
+    if isinstance(obj, (str, int, float)):
```

Comment:
You can remove this since else in L:359 should handle it, no?

Reply:
yep